Makefile

```diff
@@ -115,6 +115,14 @@ tail:
 	@echo "🥫 Reading logs (Apache2, Nginx) …"
 	tail -f logs/**/*
 
+test_cover: step_in, cover
+
+step_in:
+	@echo "🥫 Stepping into the backend container …"
+	docker exec -it po-backend-1 bash
+cover:
+	@echo "🥫 running …"
+	perl -MDevel::Cover t/*.t
```

Comment:
https://medium.com/codex/perl-test-coverage-when-you-dont-have-a-makefile-d19db387ff6e this seems a good way to integrate !

To pass an environment in docker-compose we can use the "-e" option

So it would be:

```
${DOCKER_COMPOSE_TEST} run --rm -e HARNESS_PERL_SWITCHES="-MDevel::Cover" backend prove -l
```